assets/app/views/services.html

```diff
@@ -5,46 +5,46 @@
         <h1>Services</h1>
       </div>
       <alerts alerts="alerts"></alerts>
-      <div class="table-responsive">
-        <table class="table table-bordered table-hover">
-          <thead>
-            <tr>
-              <th>Name</th>
-              <th>Cluster IP</th>
-              <th>External IP</th>
-              <th>Ports</th>
-              <th>Selector</th>
-              <th>Age</th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr ng-if="(services | hashSize) == 0"><td colspan="6"><em>{{emptyMessage}}</em></td></tr>
-            <tr ng-repeat="service in services | orderObjectsByDate : true">
-              <td><a href="project/{{projectName}}/browse/services/{{service.metadata.name}}">{{service.metadata.name}}</a></td>
-              <td>{{service.spec.clusterIP}}</td>
-              <td>
-                <span ng-if="!service.status.loadBalancer.ingress.length"><em>none</em></span>
-                <span ng-repeat="ingress in service.status.loadBalancer.ingress | limitTo: 4">{{ingress.ip}}<span ng-if="!$last">,
-                </span></span><span ng-if="service.status.loadBalancer.ingress.length === 5">,
-                  {{service.status.loadBalancer.ingress[4].ip}}</span><span ng-if="service.status.loadBalancer.ingress.length > 5">,
-                  and {{service.status.loadBalancer.ingress.length - 4}} others</span>
-              </td>
-              <td>
-                <span ng-if="!service.spec.ports.length"><em>none</em></span>
-                <span ng-repeat="portMapping in service.spec.ports | limitTo: 4">{{portMapping.port}} ({{portMapping.protocol}})<span ng-if="!$last">,
-                </span></span><span ng-if="service.spec.ports.length === 5">,
-                  {{service.spec.ports[4].port}} ({{service.spec.ports[4].protocol}})</span><span ng-if="service.spec.ports.length > 5">,
-                  and {{service.spec.ports.length - 4}} others</span>
-              </td>
-              <td>
-                <span ng-if="!service.spec.selector"><em>none</em></span>
-                <span ng-repeat="(selectorLabel, selectorValue) in service.spec.selector">{{selectorLabel}}={{selectorValue}}<span ng-show="!$last">, </span></span>              
-              </td>
-              <td><relative-timestamp timestamp="service.metadata.creationTimestamp" drop-suffix="true"></relative-timestamp></td>
-            </tr>
-          </tbody>
-        </table>
-      </div>
+      <table class="table table-bordered table-hover table-mobile">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Cluster IP</th>
+            <th>External IP</th>
+            <th>Ports</th>
```

Comment:
Fixed typo ready for merge
https://github.com/openshift/origin/pull/5087/files#diff-9d3529dae36b903d64ba7957d01e5d32